src/test/java/org/ambraproject/rhino/AbstractRhinoTest.java

```diff
@@ -1,15 +1,10 @@
 package org.ambraproject.rhino;
 
-import static org.mockito.ArgumentMatchers.any;
```

Comment:
I'm not moving these imports around myself, it seems like Eclipse and IntelliJ have different rules for optimizing imports.

I don't mind the noise too much, but it would be good to standardize this if we can.

Reply:
I agree.  :+1:   I don't mind the re-arranging of the imports.  However, I do want to keep the line lengths to <= 100 characters so that we don't have to scroll right to view the code on github =) .